install-validator/cmd/validate/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	ext "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	"k8s.io/client-go/rest"
+
+	nexuscompare "github.com/vmware-tanzu/graph-framework-for-microservices/common-library/pkg/nexus-compare"
+	"github.com/vmware-tanzu/graph-framework-for-microservices/install-validator/pkg/dir"
+	kubewrapper "github.com/vmware-tanzu/graph-framework-for-microservices/install-validator/pkg/k8s-utils"
+)
+
+const (
+	dirEnv        = "CRD_SPEC_DIR"
+	forceEnv      = "CRD_FORCE"
+	groupNamePath = "/NAME"
+)
+
+func main() {
+	directory := ""
+	if directory = os.Getenv(dirEnv); directory == "" {
+		directory = "/crds"
+	}
+	force := false
+	if crdForceStr := os.Getenv(forceEnv); strings.ToLower(crdForceStr) == "true" {
+		force = true
+	}
+
+	// setup k8s client
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		logrus.Error(err)
```

Comment:
we must panic at this point yeah. 
Observe that in most cases, we are just logging the error message and moving forward.

Reply:
yeah, obviously. Changed the errors to fatals in main